foreman/foreman.spec

```diff
@@ -540,10 +544,24 @@ cp %%{buildroot}%%{%{name}_bundlerd_dir}/%%{gem_name}.rb ./bundler.d/%%{gem_name
 unlink tmp \\
 \\
 export BUNDLER_EXT_NOSTRICT=1 \\
-/usr/bin/%%{?scl:%%{scl}-}rake %%{-r*}%%{!?-r:plugin:assets:precompile[%%{-n*}%%{!?-n:%%{gem_name}}]} RAILS_ENV=production --trace \\
+%%{?-s:/usr/bin/%%{?scl:%%{scl}-}rake %%{-r*}%%{!?-r:plugin:assets:precompile[%%{-n*}%%{!?-n:%%{gem_name}}]} RAILS_ENV=production --trace} \\
+%%{?-a:/usr/bin/%%{?scl:%%{scl}-}rake plugin:apipie:cache[%%{gem_name}] RAILS_ENV=development cache_part=resources OUT=%%{buildroot}%%{gem_instdir}/public/apipie-cache/plugin/%%{gem_name} #gem_instdir}/public/apipie-cache/plugin/%%{gem_name} --trace} \\
 \\
 popd \\
-rm -rf ./usr
+rm -rf ./usr \\
+%%{?-a:mkdir -p %%{buildroot}%%{foreman_dir}/public/apipie-cache/plugin} \\
+%%{?-a:ln -s %%{gem_instdir}/public/apipie-cache/plugin/%%{gem_name} %%{buildroot}%%{foreman_dir}/public/apipie-cache/plugin/%%{gem_name}}
+EOF
+
+cat > %{buildroot}%{_sysconfdir}/rpm/macros.%{name}-apipie << EOF
+# Common locations
+%%%{name}_apipie_cache_plugin %%{gem_instdir}/public/apipie-cache/plugin/%%{gem_name}
+%%%{name}_apipie_cache_foreman %%{foreman_dir}/public/apipie-cache/plugin/%%{gem_name}
+
+%%%{name}_apipie_cache_plugin_post \\
++cp -r %%{foreman_apipie_cache_foreman}/* %%{foreman_dir}/public/apipie-cache/ \\
```

Comment:
If there really must be two copies for some reason, and we can't remove it or symlink them, then we should copy them at build time and have the package ship them both, not copy it in %post.  This will ensure that file ownership, file cleanup, "rpm -V" etc works, and the RPM should compress away the second copy.